src/Sentry.AspNet/Sentry.AspNet.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+    <PackageId>Sentry.AspNet</PackageId>
+    <AssemblyName>Sentry.AspNet</AssemblyName>
+    <RootNamespace>Sentry.AspNet</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
```

Comment:
Nice!
Doesn't need the signature stuff? even better, maybe a PR to replace it in the whole solution all of those `AssemblyInfo.cs`?

Reply:
Nope, it only needs signature for strong named assemblies. I think it's the same for the attribute actually. In fact, this is the exact thing as applying the attribute, but using csproj instead.

Reply:
Apparently the assemblies do get strong named on CI, so I have to add the public key anyway.

Reply:
I think strong name in release builds, if not all builds.

Nice, can we have this done via csproj in all projs?